spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -64,8 +58,9 @@ public void publishMessage() {
 }
 ----
 
-The default serialization and deserialization of POJOs uses Jackson JSON via the `JacksonMessageConverter` class.
-To set a new message converter in `PubSubTemplate`, use the `setMessageConverter()` method.
+By default, the `SimplePubSubMessageConverter` is used to convert payloads of type `byte[]`, `ByteString`, `ByteBuffer`, and `String` to Pub/Sub messages.
+
+For serialization and deserialization of POJOs using Jackson JSON, configure the `PubSubTemplate` to use the `JacksonPubSubMessageConverter` by using the `setMessageConverter()` method.
```

Comment:
You tell the user to manually set their own Jackson converter here, but in the autoconfig you provide a bean for Jackson converter using an object mapper. The combination of these two implies that you want to user to provide a bean for his or her own objectmapper, but not a jackson converter, and then you want the user to get the jackson converter from the context manually and set it in the template class manually?  Why not set it up in such a way that if the user has provided an objectmapper then the jackson converter is automatically used?

Reply:
You're right, I need to add some clarification here.